hack/test/install-prometheus.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+PROMETHEUS_NAMESPACE="$1"
+PROMETHEUS_VERSION="$2"
+KUSTOMIZE="$3"
+GIT_VERSION="$4"
+
+TMPDIR="$(mktemp -d)"
+trap 'echo "Cleaning up $TMPDIR"; rm -rf "$TMPDIR"' EXIT
+
+echo "Downloading Prometheus resources..."
+curl -s "https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/refs/tags/${PROMETHEUS_VERSION}/kustomization.yaml" > "${TMPDIR}/kustomization.yaml"
+curl -s "https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/refs/tags/${PROMETHEUS_VERSION}/bundle.yaml" > "${TMPDIR}/bundle.yaml"
+
+echo "Patching namespace to ${PROMETHEUS_NAMESPACE}..."
+(cd "$TMPDIR" && $KUSTOMIZE edit set namespace "$PROMETHEUS_NAMESPACE")
+
+echo "Applying Prometheus base..."
+kubectl apply -k "$TMPDIR" --server-side
+
+echo "Waiting for Prometheus Operator pod to become ready..."
+kubectl wait --for=condition=Ready pod -n "$PROMETHEUS_NAMESPACE" -l app.kubernetes.io/name=prometheus-operator
+
+echo "Applying overlay config..."
+$KUSTOMIZE build config/overlays/prometheus | sed "s/cert-git-version/cert-${VERSION}/g" | kubectl apply -f -
+
+echo "Waiting for metrics scraper to become ready..."
+kubectl wait --for=create pods -n "$PROMETHEUS_NAMESPACE" prometheus-prometheus-0 --timeout=60s
+kubectl wait --for=condition=Ready pods -n "$PROMETHEUS_NAMESPACE" prometheus-prometheus-0 --timeout=120s
+
+echo "Prometheus deployment completed successfully."
```

Comment:
Can we give a space here . Otherwise, LGTM

Reply:
Done!